pkg/cli/subcommands/check/command.go

```diff
@@ -217,6 +219,8 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 				fx.Provide(func() optional.Option[rcserviceha.Component] { return optional.NewNoneOption[rcserviceha.Component]() }),
 
 				getPlatformModules(),
+				jmxloggerimpl.Module(),
+				fx.Supply(jmxloggerimpl.NewDisabledParams()),
```

Comment:
I'm guessing the reason for `fx.Supply` instead of `fx.Provide(jmxloggerimpl.NewDisabledParams)` is that Supply is preferred for pure data values, by convention. Is that correct?

Reply:
Yes.
Both options are OK, `fx.Supply` takes an existing instance whereas `fx.Provide` takes a factory.